openmetadata-ui/src/main/resources/ui/src/utils/ToastUtils.ts

```diff
@@ -44,6 +45,7 @@ export const showErrorToast = (
     }
   }
   toast.error(errorMessage, {
+    ...defToastOptions,
```

Comment:
@darth-coder00 Instead of defining here for each type of toast, we can add `hideProgressBar={true}` to `<ToastContainer/>` in App.tsx